packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -276,47 +239,161 @@ function ListView() {
   }
 
   return (
-    <Wrapper>
+    <Layout>
       <PageTitle name="Webhooks" />
-      <Header {...headerProps} />
-      {canRead && (
-        <div className="list-wrapper">
-          {rowsCount > 0 ? (
-            <List
-              {...listProps}
-              customRowComponent={props => {
-                return (
-                  <ListRow
-                    {...props}
-                    canUpdate={canUpdate}
-                    canDelete={canDelete}
-                    onCheckChange={handleChange}
-                    onEditClick={handleGoTo}
-                    onDeleteCLick={handleDeleteClick}
-                    onEnabledChange={handleEnabledChange}
-                    itemsToDelete={webhooksToDelete}
-                  />
-                );
-              }}
-            />
-          ) : (
-            <EmptyState
-              title={formatMessage({ id: 'Settings.webhooks.list.empty.title' })}
-              description={formatMessage({ id: 'Settings.webhooks.list.empty.description' })}
-              link="https://strapi.io/documentation/developer-docs/latest/development/backend-customization.html#webhooks"
-              linkText={formatMessage({ id: 'Settings.webhooks.list.empty.link' })}
-            />
+      <Main labelledBy="webhooks">
+        <HeaderLayout
+          as="h1"
+          id="webhooks"
+          title={formatMessage({ id: 'Settings.webhooks.title' })}
+          subtitle={formatMessage({ id: 'Settings.webhooks.list.description' })}
+          primaryAction={(
+            <Button onClick={() => (canCreate ? handleGoTo('create') : {})} startIcon={<AddIcon />}>
+              {formatMessage({ id: 'Settings.webhooks.list.button.add' })}
+            </Button>
           )}
-          <ListButton>{canCreate && <Button {...omit(newButtonProps, 'Component')} />}</ListButton>
-        </div>
-      )}
+        />
+
+        <ContentLayout>
+          <>
+            {rowsCount > 0 ? (
```

Comment:
Something is wrong with the table, I can't focus the delete icon in each row and the last row also (with the keyboard).

![webhooksgif](https://user-images.githubusercontent.com/7756284/129576260-adf01a18-0044-4189-a5a2-52f178719c7c.gif)

Reply:
You can focus on it if you are already focused on the edit icon then using tab. I talked about it with Marvin, if I understood well it's because it's a button/action so you have to use tab to access it 🕵️‍♀️

Reply:
Oh thanks for the button, indeed it is better with the tab. But, I can't reach the last list row with the keyboard 😢

Reply:
(rows should include the header)

Reply:
Oh right same here, I'm not sure why though! @mfrachet do you have an idea? 🤕

Reply:
Fixed 🔥